### ISSUE TITLE
[Bug]: Netdata process using 200% CPU on

### ISSUE THREAD

**Issue by GoAdvisedGroup** [netdata/netdata#12442]
### Bug description

When I got to the office this morning I noticed a higher load then normal on one of our cloud servers. After further inspection it seems that Netdata is the culprit. 
![image](https://user-images.githubusercontent.com/83397704/158789315-3ff5f7da-05a1-477c-8970-c9818ab7164c.png)
Somehow it is using 2 full cores on our server.
I tried to update netdata but it was already the most recent version. So after that I tried a netdata reinstall, still the same issue. Our other cloud servers are running the same configuration, hardware, software and Netdata version(v1.33.1-189-nightly).

Is there some way that I could debug netdata to see what is taking so much CPU.

### Expected behavior

Low CPU usage

### Steps to reproduce

None

### Installation method

kickstart.sh

### System info

```shell
/etc/cloudlinux-release:CloudLinux release 8.5 (Vladimir Shatalov)
/etc/os-release:NAME="CloudLinux"
/etc/os-release:VERSION="8.5 (Vladimir Shatalov)"
/etc/os-release:ID="cloudlinux"
/etc/os-release:ID_LIKE="rhel fedora centos"
/etc/os-release:VERSION_ID="8.5"
/etc/os-release:PLATFORM_ID="platform:el8"
/etc/os-release:PRETTY_NAME="CloudLinux 8.5 (Vladimir Shatalov)"
/etc/os-release:ANSI_COLOR="0;31"
/etc/os-release:CPE_NAME="cpe:/o:cloudlinux:cloudlinux:8.5:GA:server"
/etc/plesk-release:18.0.42.0
/etc/plesk-release:Plesk Obsidian 18.0
/etc/redhat-release:CloudLinux release 8.5 (Vladimir Shatalov)
/etc/system-release:CloudLinux release 8.5 (Vladimir Shatalov)
```

### Netdata build info

```shell
Features:
    dbengine:                   YES
    Native HTTPS:               YES
    Netdata Cloud:              YES
    ACLK Next Generation:       YES
    ACLK-NG New Cloud Protocol: YES
    ACLK Legacy:                NO
    TLS Host Verification:      YES
    Machine Learning:           YES
    Stream Compression:         NO
Libraries:
    protobuf:                YES (bundled)
    jemalloc:                NO
    JSON-C:                  YES
    libcap:                  NO
    libcrypto:               YES
    libm:                    YES
    tcalloc:                 NO
    zlib:                    YES
Plugins:
    apps:                    YES
    cgroup Network Tracking: YES
    CUPS:                    NO
    EBPF:                    NO
    IPMI:                    NO
    NFACCT:                  NO
    perf:                    YES
    slabinfo:                YES
    Xen:                     NO
    Xen VBD Error Tracking:  NO
Exporters:
    AWS Kinesis:             NO
    GCP PubSub:              NO
    MongoDB:                 NO
    Prometheus Remote Write: NO
```

**Comment by gert28** [comment]
I see the same issue on this version

**Comment by vkalintiris** [comment]
Can you share any non-default configuration options that you might be using in `netdata.conf`?

> So after that I tried a netdata reinstall, still the same issue

Curious, is the agent's dashboard still accessible while on high CPU usage?

**Comment by ilyam8** [comment]
Some good news:
 - it is not 200%, but 100% 😅 
 - the problem is found, will apply the fix and republish nightly packages asap.

**Comment by gert28** [comment]
Good to hear that this is solved. But how can I fix this on my install? I just have to wait for the next release?

**Comment by ilyam8** [comment]
@gert28 You need to update Netdata

```bash
# as root user
/opt/netdata/usr/libexec/netdata/netdata-updater.sh
# or (depends on your install install prefix)
/usr/libexec/netdata/netdata-updater.sh
```

**Comment by taha-yassine** [comment]
I don't why but for some reason my netdata running on docker is stuck on v1.33.1-189-nightly so one version behind the build that comes with the fix. I checked the docker image and it is up to data. Maybe something went wrong with the docker build job. So I guess I'll have to wait for a new image to be pushed before I can fix my buggy version.

**Comment by ilyam8** [comment]
@taha-yassine the latest published docker image is v1.33.1-195-nightly

```bash
~ $ docker pull netdata/netdata; docker run -d --rm --name netdata netdata/netdata; docker exec netdata netdata -v; docker stop netdata
Using default tag: latest
latest: Pulling from netdata/netdata
Digest: sha256:88bbec664f221567e23c88ad7df079937c34c126c74dfd8c58754571b975e0c3
Status: Image is up to date for netdata/netdata:latest
docker.io/netdata/netdata:latest
28ce8bd0a32d50dcbc39affcc1ac5325d8b1ba55ea97bd1ef5567b84af76d2e3
netdata v1.33.1-195-nightly
netdata
```

**Comment by taha-yassine** [comment]
@ilyam8 you are right, for some reason docker-compose wasn't pulling the latest image. Thanks for your help!
